src/config/config.ts

```diff
@@ -0,0 +1,10 @@
+import path from "path";
+import os from "os";
+
+const tmpPath = path.join(os.tmpdir(), ".mouji");
```

Comment:
/uj
if I want to read these it's gonna be hard to find them, maybe there's a conventional path for application state like `/var/lib` in unix that would be possible to use. just don't put it in the home dir buckaroo

or maybe idgaf if I can find them bc I don't care

/rj
`apps/server/src/server/apps/src/__server__/__mocks__/moujipath/moujidir/moujistate/mouji` 🤓

Reply:
I think the home dir is closest to conventional when it comes to stuff like this. Another approach would be to save the file to the CWD, and let the user specify the file name when doing a replay.

Reply:
I did the CWD thing. I think we need some tests though.